views/deployment-config.html

```diff
@@ -9,23 +9,24 @@
     <dd>{{resource.metadata.creationTimestamp | date:'medium'}}</dd>
   </dl>
   <h3>Triggers</h3>
-  <dl class="dl-horizontal" ng-repeat="trigger in resource.triggers">
+  <dl class="dl-horizontal" ng-repeat="trigger in resource.spec.triggers">
     <dt>Type</dt>
     <dd>{{trigger.type}}</dd>
     <dt ng-if-start="trigger.type == 'ImageChange'">Image</dt>
+    <!-- FIXME: Need to verify this in v1beta3. -->
     <dd ng-if-end>{{trigger.imageChangeParams.from.name}}:{{trigger.imageChangeParams.tag}}</dd>
   </dl>
   <h3>Replication Settings</h3>
   <dl class="dl-horizontal">
     <dt>Replicas</dt>
-    <dd>{{resource.template.controllerTemplate.replicas}}</dd>   
+    <dd>{{resource.spec.replicas}}</dd>
   </dl>
   <h4>Selector</h4>
   <dl class="dl-horizontal">
-    <dt ng-repeat-start="(selectorKey, selectorValue) in resource.template.controllerTemplate.replicaSelector">{{selectorKey}}</dt>
+    <dt ng-repeat-start="(selectorKey, selectorValue) in resource.spec.selector">{{selectorKey}}</dt>
     <dd ng-repeat-end>{{selectorValue}}</dd>
-  </dl>  
-  <kubernetes-object-describe-pod-template template="resource.template.controllerTemplate.podTemplate.desiredState.manifest"></kubernetes-object-describe-pod-template>  
+  </dl>
+  <kubernetes-object-describe-pod-template template="resource.spec.template.spec"></kubernetes-object-describe-pod-template>
```

Comment:
@ironcladlou can you confirm that `deploymentConfig.spec.template` is going away in favor of `deploymentConfig.spec.templateRef` once v1beta1 does?